dropwizard-migrations/src/main/java/io/dropwizard/migrations/DbCommand.java

```diff
@@ -50,8 +60,15 @@ public void configure(Subparser subparser) {
 
     @Override
     public void run(Namespace namespace, Liquibase liquibase) throws Exception {
+        String scopeId = null;
+        if (scopedObjects != null) {
+            scopeId = Scope.enter(scopedObjects);
+        }
         final AbstractLiquibaseCommand<T> subcommand =
             requireNonNull(subcommands.get(namespace.getString(COMMAND_NAME_ATTR)), "Unable find the command");
         subcommand.run(namespace, liquibase);
+        if (scopeId != null) {
+            Scope.exit(scopeId);
+        }
```

Comment:
Would it make sense to put at least the `subcommand.run(...)` statement in a `try` block and exit the scope in the `finally` block?

Reply:
I think, that this is a good idea. Otherwise a further call to the exit method will probably fail, because the scope id wouldn't be matching then